src/Symfony/Component/Form/Tests/AbstractDivLayoutTest.php

```diff
@@ -453,7 +453,7 @@ public function testNestedFormError()
         $form = $this->factory->createNamedBuilder('name', 'form')
             ->add($this->factory
                 ->createNamedBuilder('child', 'form', null, array('error_bubbling' => false))
-                ->add('grandChild', 'form')
+                ->add('grandChild', 'text')
```

Comment:
Ok it was reverted when I tried on master because of the conflict resolution. So I think you should not make that change and fix the layout test in this case because it does impact `form_rest` and this should be tested too.